conf.py

```diff
@@ -50,12 +50,17 @@
 
 
 html_theme_options = {
-  "github_url": "https://github.com/username/",
+  "github_url": "https://github.com/KaddenG52",
   "twitter_url": "https://twitter.com/username",
-  "search_bar_text": "Search this site...",
+  "search_bar_text": "Looking For Something...",
 #   "navbar_end": ["search-field.html"],
 }
 
+html_theme_options = {
+   ...
+   "announcement": "Here's a <a href='https://pydata.org'>PyData Announcement!</a>",
+}
```

Comment:
```suggestion
  "search_bar_text": "Looking For Something...",
#   "navbar_end": ["search-field.html"],
   "announcement": "Here's a <a href='https://pydata.org'>PyData Announcement!</a>",
}
```

what you had done was create the variable with the values you wanted then overwrite it with a new value. 

```
html_theme_options = {
   ...
```
was supposed to say to add the `"announcement"` line to your existing `html_theme_options` variable. the `...` was to stand in for whatever other things you already had there.